columnflow/tasks/cutflow.py

```diff
@@ -307,21 +307,35 @@ def requires(self):
             for d in self.datasets
         }
 
+    def plot_parts(self) -> law.util.InsertableDict:
+        parts = super().plot_parts()
+        parts["category"] = self.branch_data
```

Comment:
is there a reason to remove the "cat_" prefix?
```suggestion
        parts["category"] = f"cat_{self.branch_data}"
```